### PR TITLE
Semantic markup of footer menus

### PR DESCRIPTION
**Pull request by DJMyles**
## Description

See https://github.com/AusDTO/gov-au-ui-kit/issues/448 for background.

TL;DR - move the `<h2>` out of the `<ul>` and wrap the `<h2>` and `<ul>` in a `<div class="footer-menu">`, then extend the grid out to the `.footer-menu` element

## Definition of Done

- [ ] Content/documentation reviewed by Julian or someone in the Content Team
- [ ] UX reviewed by Gary or someone the Design team
- [ ] Code reviewed by one of the core developers
- [ ] Acceptance Testing
  - [ ] [Cross-browser tested against standard browser matrix](https://docs.google.com/spreadsheets/d/1B8pGWSiFbWhurnDISvD2MKnI0IF_T8tU2sl1naZCw1E/edit#gid=1030964576)
  - [ ] Tested on multiple devices (TBD)
  - [ ] HTML5 validation (CircleCI)
  - [ ] Accessibility testing & WCAG2 compliance (`npm test`)
- [ ] Stakeholder/PO review
- [ ] CHANGELOG updated

### CODE REVIEW

**Comment by klepas** [comment]
Things I need to do to merge this:

- update the template/markup example
- consider contacting teams/implementers re. the markup changes
- update guidance as applicable on the footer feature

**Comment by DJMyles** [comment]
@klepas silly question but are we able to build scss changes out to the template / markup example as part of commit or is that something you guys do on your end?

**Comment by klepas** [comment]
@DJMyles —

> silly question but are we able to build scss changes out to the template / markup example as part of commit or is that something you guys do on your end?

We update the eg. markup in tandem. (:

I did and created a new PR for it: https://github.com/AusDTO/gov-au-ui-kit/pull/453
